server/endPoints-file/all_modules_pageEndPoint.js

```diff
@@ -0,0 +1,79 @@
+const db = require("../database/db");
+
+const getAllDataForAllModulesPage = async (req, res) => {
+  // +DISTINCT ON (modules.name)
+  const query = `SELECT 
+  modules.id AS module_id,
+  modules.name AS module_name,
+  modules.description AS module_description,
+  topics.id AS topic_id,
+  topics.module_id,
+  topics.topic_name,
+  topics.description AS topic_description,
+  topics.reference_link,
+  topics.test_link
+  FROM topics
+  INNER JOIN modules
+  ON modules.id = topics.module_id
+  `;
+  // const query = `
```

Comment:
Could you please delete the commented code if you are not going to use it? Thank you

Reply:
this query is here for future inpruvement